sources/translations/en/settings.json

```diff
@@ -16,6 +16,13 @@
                 "disable": "Disable hiding window to the system tray functionality."
             }
         },
+                "custombar": {
+                    "name": "Custom window bar",
+                    "description": "Uses non-native window bar. To access settings press title of window. <b>Important:</b> it injects custom CSS. This might potentially let Discord to know it is modded.",
+                    "labels": {
+                        "use": "Use custom window bar. (Requires restart)"
+                    }
+                },
```

Comment:
I would accept this if it would describe more about HTML injection. This is because it is quite easy to watch for the changes with HTML, but it is harder to verify the CSS. And since you're injecting CSS without HTML, Discord would both have a hard time to discover your changes on that side. However, nothing stops it from adding the right `querySelector` to find out about the WebCord's non-native taskbar and take control on it (do all actions that the user can do with it). This is the reason why I don't like this description and I would like to have this listed in *Advanced* section.

Reply:
Done